tools/metaquantome/macros.xml

```diff
@@ -0,0 +1,51 @@
+<macros>
+    <token name="@VERSION@">1.0.0</token>
+    <token name="@GVERSION@">1.0.0-0</token>
```

Comment:
`<token name="@GVERSION@">@VERSION@-0</token>`

Reply:
hm, that doesn't seem to work (the @VERSION@ in the token isn't parsed)